environment/dev/app/php.ini

```diff
@@ -1,5 +1,7 @@
 [PHP]
 memory_limit = 256M
+upload_max_filesize = 3M
+post_max_size = 3M
```

Comment:
Why exactly 3?
Is it related to uploaded files?
Maybe we should increase this a little bit more?